### ISSUE TITLE
"memory card removed" message when vita sleeps

### ISSUE THREAD

**Issue by jubilees** [CelesteBlue-dev/PSVita-StorageMgr#30]
hello, O have recently installed the latest henkaku to my vita and it was fine until I used the save data manager and after that I started getting" memory card removed" message everytime it sleeps.

**Comment by CelesteBlue-dev** [comment]
Please send ur0:tai/storagemgr_logs.txt

**Comment by CelesteBlue-dev** [comment]
Please test : https://github.com/CelesteBlue-dev/PSVita-StorageMgr/releases/tag/v2.1b1

**Comment by jubilees** [comment]
Is this for SD2Vita or is it applicable to the sony 64gb memory card? Sorry for the noon questions, I'm fairly new to this.

**Comment by CelesteBlue-dev** [comment]
you copy storagemgr.skprx and storage_config.txt to ur0:tai and add the .skprx line in config.txt
Then you  can configure for exemple MCD=uma0 to mount official memcard to uma0:.

**Comment by jubilees** [comment]
Ok, I copied them both to it. How can I configure?

**Comment by jubilees** [comment]
Also how do I add it to config.txt?

**Comment by CelesteBlue-dev** [comment]
in ur0:tai/config.txt under *KERNEL
add : 
ur0:tai/storagemgr.skprx
in ur0:tai/storage_config.txt
change MCD=uma0 by any mount point you want
for exemple you can do : 
UMA=ux0
GCD=grw0
MCD=xmc0

**Comment by jubilees** [comment]
Do I have to change the mount point or can I leave it as it is?

**Comment by CelesteBlue-dev** [comment]
as you want, what do you want to mount as ux0 ? sd2vita ? memorycard ? psvsd ?

**Comment by jubilees** [comment]
Memory card

**Comment by jubilees** [comment]
Hey mate! So I did it all like you said and the message no longer pops up! Thanks a lot!

**Comment by jubilees** [comment]
Hello, so I received the message again after leaving a game open and letting the vita fall into sleep mode.

**Comment by CelesteBlue-dev** [comment]
you use v2.1 ?
please make a video

**Comment by jubilees** [comment]
Yes I did. I was also testing it out yesterday and the ps vita doesn't show the message anymore if there is no game cartridge inside the slot.

**Comment by CelesteBlue-dev** [comment]
"I was also testing it out yesterday and the ps vita doesn't show the message anymore if there is no game cartridge inside the slot."
This is an issue where StorageMgr treats official cartridge as a SD2VITA...
to fix...

**Comment by itsminht** [comment]
This has also just started happening to me. Although I have only encountered this once so far (first time this has happened ever actually)- I leave my Vita on sleep mode and the next morning when I switch it on it says "Memory card has been removed etc.". Not sure if this makes a difference but I should add that when I left my Vita on sleep mode it was during a game left open I'm running storagemgr 3.0 plugin

@CelesteBlue-dev Is this issue still present as of v3.0? Any ideas what might cause this? Not sure if this is an issue with this plugin or with gamesd.skprx as well...

@jubilees Does this still happen to you or not anymore? Also what did you set up your storagemgr plugin as? Like is your sd2vita mounted as uma0 or ux0? Concerned because I researched online that this usually happens before your memory card dies out but now I can't tell if it's happening because of this plugin or memory card is actually about to die

**Comment by jubilees** [comment]
@itsminht  No it doesn't happen anymore to my ps vita. I do not have an sd2vita, I just have a game cartridge inside the slot and while I was checking my vita I had noticed that when it wasn't inside the vita the message wouldn't pop until 3 days later. The only thing I did was what celeste mentioned to do up earlier in the thread. After I did it, the message stopped popping up only to come back a few days later. I didn't do anything else after and  it doesn't pop anymore.

**Comment by itsminht** [comment]
Ahh okay, but why would you use the storagemgr plugin for your ps vita if you don't have sd2vita? So like you only use sony memory card right? I stumbled across your reddit post and was in a similar situation to yours in regards to the error message.

**Comment by jubilees** [comment]
Honestly, I have no idea, that's why early in this thread I had put that I don't use sd2vita but celeste told me do what was stated above. I hear you man, I found no one when I was looking for the solution. Until someone told me to submit an issue in github. Hopefully if you do what I did, the message won't pop anymore.
